MyFirstApplication/Extensions/Pagination.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MyFirstApplication.Models;
+
+namespace MyFirstApplication.Extensions
+{
+    public static class Extensions
```

Comment:
Class name should be more specific, like `ListExtensions`, or `ControllerExtensions`, depending on which type of object it applies to represented by `this` in the Pagination method. That way you can have more than one extension methods that apply to a list in one class. Create a separate class if you like to apply extension methods to a controller.